config/ui-functions/mimic3-cache

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+CACHE_DIR=${CACHE_DIR:-/vx/ui/mimic-cache}
+mkdir -p ${CACHE_DIR}
+
+switch=$1
+input=$2
+
+if [[ $switch != "-w" ]]; then
+    exit 0
+fi
+
+# trim leading and trailing spaces
+input=$(echo "$input" | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//')
+
+# have we cached this?
+hash=`echo "${input}" | sha1sum | cut -d' ' -f1`
+filename="${CACHE_DIR}/${hash}.wav"
+
+# if no file or the file is empty (can happen on crash in previous attempts to generate this file), generate it
+if ! [ -s "${filename}" ]; then
+    echo "${input}" | /usr/bin/mimic3 --remote --voice en_US/m-ailabs_low --stdout > ${filename}
```

Comment:
These never get cleaned up, right? Seems like a LRU cache would be in order, or perhaps a cache that gets cleared on election unconfigure.

Reply:
indeed, one of my next quick-and-dirty cleanup ideas that I want to add is, in mimic3-cache, simply remove any files that are older than 2 months.

Reply:
Also, not the `--remote`, that's the part that makes `mimic3` contact the http server instead of loading everything itself.